packages/SwingSet/misc-tools/extract-transcript-from-slogfile.js

```diff
@@ -71,8 +71,8 @@ async function run() {
       }
       case 'deliver-result': {
         console.log(` -- deliver-result`);
-        const entry = { transcriptNum, d: delivery, syscalls };
         transcriptNum += 1;
+        const entry = { transcriptNum, d: delivery, syscalls };
```

Comment:
BTW I'd love to find a clean way to keep the `transcriptNum` in the DB, to avoid potential drift or fencepost errors with this external incrementing counter.

Reply:
It is in the DB already. Here it's from the slog, which doesn't include the transcript pos, so it has to be regenerated. It would indeed be nice to include this info in the slogfile.